apps/server-asset-sg/src/features/favorites/favorites.controller.ts

```diff
@@ -23,12 +25,14 @@ export class FavoritesController {
   }
 
   @Put('/:assetId')
```

Comment:
So users can potentially add assets to theri favourits which they are not allowed to see because of lacking workgroup access? Would this be a loophole to view assets if (of course only if i know the id despite not being allowed to view it)

Reply:
Favorites are not yet correctly authorized as they are not really implemented. If we ever get to them, we would probably write a new policy just for them.